src/browser.ts

```diff
@@ -69,6 +81,7 @@ export const launchBrowser = async (
       '--disable-search-engine-choice-screen',
       `--proxy-server=http://localhost:${appSettings.proxy.port}`,
       `--ignore-certificate-errors-spki-list=${certificateSPKI}`,
+      `--load-extension=${extensionPath}`,
```

Comment:
This will be put behind a feature flag that can be toggled via settings in a future PR.